IM_calculation/scripts/calculate_ims.py

```diff
@@ -163,19 +191,42 @@ def main():
     if "FAS" in im:
         im_options["FAS"] = calc.validate_fas_frequency(args.fas_frequency)
 
+    advanced_im_config = advanced_IM_factory.advanced_im_config(
+        args.advanced_ims, args.advanced_im_config, args.OpenSees_path
+    )
+
     # Create output dir
     utils.setup_dir(args.output_path)
     if not args.simple_output:
         utils.setup_dir(os.path.join(args.output_path, calc.OUTPUT_SUBFOLDER))
 
+    if args.observed != None:
```

Comment:
I'm not convinced this is the best way to reduce the station's calculated. I guess this is a sandboxed argument and it would work.

Reply:
I've been thinking about it and it's not inline with keeping structure / complexity out of calculation scripts. 

If you need that you should write a separate script that does this and then pass the output as an argument into -s (--stations)

Reply:
the only cons of doing so it that the execute command in slurm/pbs/etc can be extremely long.
(lets say we have a matching 80 stations)

Reply:
I don't personally see that as a big deal. Previously we've ignored the station matching because IM_calc didn't use much resources but can definitely see this being an issue for Adv_IM.

Reply:
i'll remove this from calulate_im for now.
but if we starting to have issue with the length of sysargv, we can always change this to read a tmp/station_list_file thats generated on the fly.